src/api-specification/peer-controller/retrieving-user-details.md

```diff
@@ -0,0 +1,99 @@
+# Retrieving user details
+
+> Retrieves all of the details about the [authorized FIB user](../auth-controller/authorizing-a-user-in-the-system.md).
+
+**Protocol**: `HTTP`
+
+**Encoding**: `JSON`
+
+**Endpoint**: `/data/api/v1/peer-management/account`
```

Comment:
I believe the endpoint is wrong here. It should be "auth" instead of "data": auth/api/v1/peer-management/account.